tests/unit/horizontal-bar-test.js

```diff
@@ -96,3 +99,23 @@ test("Margin bottom depends on the label padding and title offset",
     'Margin bottom is depedent on the xTitleVerticalOffset when there is x ' +
         'Axis title');
 });
+
+test("Value/Grouping Labels appear on the left/right when all data is 0 or negative",
+    function(assert) {
+
+  const component = this.subject({
+    data: all_negatives,
+  });
+
+  this.render();
+
+  const barGroups = component.$('g.bar');
+  assert.expect(barGroups.length);
+
+  barGroups.each(function(index, group) {
+    const groupLabel = $('text.group', group);
+    const valueLabel = $('text.value', group);
+    assert.ok(valueLabel.offset().left < groupLabel.offset().left);
```

Comment:
These need assertion messages

Also, come to think of it, this probably isn't the best way to do it. Using something like `_.all` would be good. 
Something like 
```
const allLabelsCorrectlyPositioned _.all(barGroups, function(group) {
  const groupLabel = $('text.group', group);
  const valueLabel = $('text.value', group);
  return valueLabel.offset().left < groupLabel.offset().left;
});
ok allLabelsCorrectlyPositioned, 'The value labels are all to the left of the group labels'
```